propagation/usecases.md

```diff
@@ -0,0 +1,167 @@
+
+### Scenarios
+
+* A project has developed a workflow using non-APEx-compliant technology and seeks to convert it into an openEO User
+  Defined Process (UDP) or OGC Application Package.
+
+### Examples
+
+* The project has created an EO data processing pipeline for generating a new air quality index and wants to encapsulate
+  it in a UDP or OGC Application Package to facilitate easy execution by stakeholders and
+  integration into a dashboard.
+
+### Solution
+
+APEx offers a set of initial [guidelines](../interoperability/algohosting.md) and
+online [resources](index.qmd#how-to-build-an-on-demand-service)
+to help projects with integrating their processing workflow into an APEx-compliant technology. Should additional
+assistance be required, the [APEx Algorithm Porting](porting.md) service can provide further support. After an initial
+analysis, APEx experts will propose an appropriate APEx-compliant technology and assist your team in its implementation.
+
+:::{.callout-tip}
+
+# Want to share your algorithm with the APEx community?
+
+Please refer to Use Case 4 to learn how APEx can support you in sharing your algorithm with the community.
+:::
+
+### Outcome
+
+The algorithm is successfully transformed and is now available as an openEO UDP or OGC Application Package.
+
+## Use Case 4: Make your algorithm available in the APEx Algorithm Catalogue
+
+### Scenarios
+
+* A project has implemented its algorithm as an openEO User Defined Process (UDP) or packaged it as an OGC Application
+  Package and wants to make it available to the APEx community.
+
+::: {.callout-tip}
+
+# Your algorithm is not yet available as an openEO UDP or packaged as an OGC Application Package?
+
+If your algorithm has not yet been implemented in an APEx-compliant technology, please refer to the previous use case to
+discover how APEx can provide support.
+:::
+
+### Examples
+
+* The project has created an openEO UDP or OGC Application Package that calculates a new air quality index and wants to
+  share this service with the EO community.
+* The project has developed a new machine learning model encapsulated in an openEO UDP or Application Package and wants
+  to ensure it runs stably and within a controlled cost range.
+* The project has developed an OGC Application Package and is seeking support to host it on an APEx-compliant platform.
+
+### Solution
+
+The [APEx Algorithm Onboarding](onboarding.md) service offers comprehensive support for integrating your algorithm into
+the APEx Algorithm Catalogue, ensuring its availability for reuse within the EO community. The service includes:
+
+* Assistance in selecting an APEx-compliant algorithm hosting platform to guarantee that your algorithm is available for
+  execution.
+* Provision of tools for automated testing and benchmarking of your algorithm.
+* Support and resources for registering your algorithm in the APEx Algorithm Catalogue.
+
```

Comment:
I thought that only algorithms deployed on some platform are entitled for entering the APEx catalogue. The solution drops the step of finding a platform, making an agreement with the platform, deploy and test it on the platform. We currently observe how complicated even the initiation of that is on the example of OpenSarToolkit.

Reply:
I think part of the solution is to separate the description for openEO and AP.
For openEO, we already have a full description online:
https://esa-apex.github.io/apex_documentation/udp_writer_guide.html#sec-udp-integration
It's also not at all a very hard thing to do, because the provider starts from something that is already running on a hosting platform.
The equivalent to openEO is an OGC API Process available via REST API, so that can be treated in a similar manner.

Deploying an OGC AP on a hosting platform, and thus setting up an OGC Process is perhaps a step that either requires a separate service from APEx or the hosting platforms. My personal understanding was always that this is a very normal and easy thing, supported by the ADES rest api, but this might be wrong. (Why is it so difficult actually?)

Reply:
As it is described right now, deploying of the OGC AP on a hosting platform (section: Hosting Platform Onboarding Support) is part of the onboarding service. However, for an openEO UDP, this step is not relevant. If we agree to support this step, mostly from an OGC AP point of view, we indeed need to align this text so the first bullet point includes the actual onboarding or hosting of the algorithm.